src/haddock/modules/analysis/clustrmsd/defaults.yaml

```diff
@@ -0,0 +1,41 @@
+criterion:
+  default: "maxclust"
+  type: "string"
+  minchars: 0
+  maxchars: 100
+  title: Criterion for fcluster
+  short: No short description yet
+  long: No long description yet
```

Comment:
I'd try to add this in this PR still

Reply:
do you mean the short and long descriptions?

Reply:
yes